<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CImageLayerDialog</class>
 <widget class="QWidget" name="CImageLayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CImageLayerDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_layer">
       <property name="styleSheet">
        <string notr="true">color: rgb(249, 133, 0);</string>
       </property>
       <property name="text">
        <string>Layer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_layerName">
         <property name="text">
          <string>LayerName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_dirveType">
         <property name="text">
          <string>DriveType</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit_layerName"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="comboBox_driveType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>gdal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>flatgis</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>agglite</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>agglitesx</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>arcgis</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bgc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>colorramp</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>debug</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>google</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>landuse</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mbtiles</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>osg</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>skyview</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tdt</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tilecache</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>vpb</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>wms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>xyz</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>54</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(44,80,114);</string>
           </property>
           <property name="title">
            <string>图层类型</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QRadioButton" name="radioButton_Image">
              <property name="styleSheet">
               <string notr="true">border:none;</string>
              </property>
              <property name="text">
               <string>影像</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_Ele">
              <property name="styleSheet">
               <string notr="true">border:none;</string>
              </property>
              <property name="text">
               <string>高程</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>54</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(44,80,114);</string>
           </property>
           <property name="title">
            <string>数据类型</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QRadioButton" name="radioBtn_singleData">
              <property name="styleSheet">
               <string notr="true">border:none;</string>
              </property>
              <property name="text">
               <string>SingleData</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioBtn_sliceData">
              <property name="styleSheet">
               <string notr="true">border:none;</string>
              </property>
              <property name="text">
               <string>SliceData</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_path">
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_path"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_find_path">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_display">
       <property name="styleSheet">
        <string notr="true">color: rgb(249, 133, 0);</string>
       </property>
       <property name="text">
        <string>Display</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox_opacityLayer">
       <property name="text">
        <string>OpacityLayer</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_opacityDegree">
       <property name="text">
        <string>OpacityDegree</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hSlider_opacityDegree">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_opacityDegreeData">
       <property name="minimumSize">
        <size>
         <width>46</width>
         <height>19</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>46</width>
         <height>19</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid rgb(0,160,234);</string>
       </property>
       <property name="text">
        <string>0.5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QCheckBox" name="checkBox_cache">
       <property name="styleSheet">
        <string notr="true">color: rgb(249, 133, 0);</string>
       </property>
       <property name="text">
        <string>SetCache</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_cacheName">
         <property name="text">
          <string>CacheName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_cachePath">
         <property name="text">
          <string>CachePath</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLineEdit" name="lineEdit_cacheName"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLineEdit" name="lineEdit_cachePath"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_find_cachePath">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ok">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/images.qrc"/>
 </resources>
 <connections/>
</ui>
